website/static/js/contribManager.js

```diff
@@ -113,7 +113,7 @@ var ContributorModel = function(contributor, currentUserCanEdit, pageOwner, isRe
             name: self.fullname
         };
         $osf.postJSON(
-            nodeApiUrl + 'beforeremovecontributors/',
+            nodeApiUrl + 'beforeremovecontributors/', // jshint ignore:line
```

Comment:
We should probably just fix this error. `nodeApiUrl` is a global that gets set in `project_base.mako`. However, we will remove it in the future in favor of using the namespaced variables on `window.contextVars`. Currently, `nodeApiUrl` is equivalent to `window.contextVars.node.urls.api`.